### DISCUSSION TITLE
fetchOrCreateMany returns invalid SQL syntax

### DISCUSSION THREAD

**Discussion by hartwm** [adonisjs/core#1753] [General]
I am getting an error 
`ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax `

I am using:
```
 "@adonisjs/core": "^5.0.0-preview-rc-1.12", 
 "@adonisjs/lucid": "^8.4.4",
 "mysql": "^2.18.1",
```

Here is the call being made
```
select * from `brands` where `brand_id` in ["74","75","296"]
```

A working call would use (). Example below.
```
select * from `brands` where `brand_id` in ("74","75","296")
```

**Comment by thetutlage** [comment]
Can you share the code you have written that I can use to reproduce the issue? We have numerous tests running against MySQL so it is less likely an issue in the package

**Reply by hartwm** [reply]
Here is an abbreviated example (not by much)
The `getWheelBrands` is just running an API query from 3rd party data and returning it as shown in the example below. There are no issues with this.

```
import WheelBrand from 'App/Models/WheelBrand' 
import { getWheelBrands } from 'Utils/WheelApi'

export default class Scrape {
  public async wheelBrands() {
    let { data } = await getWheelBrands()

//
// Example 'data' return =  [{"brandID":"74","brandName":"2 Crave Alloys","isActive":true},{"brandID":"75","brandName":"2 Crave Extreme","isActive":true}]
//

    const brands = await WheelBrand.updateOrCreateMany('brandID', data)
    return brands
 }
}
```

**Reply by hartwm** [reply]
There was an issue with my API returning a string and the Model a number. Everything is working
